sleap_chrome_remote_desktop/README.md

```diff
@@ -0,0 +1,154 @@
+
+```bash
+docker run -v /path/on/host:/path/in/container [other options] image_name [command]
+```
+
+**Notes:**
+
+- `-it` ensures that you get an interactive terminal. The `i` stands for interactive, and `t` allocates a pseudo-TTY, which is what allows you to interact with the bash shell inside the container.
+- The `-v` or `--volume` option mounts the specified directory with the same level of access as the directory has on the host.
+- The `--rm` flag in a docker run command automatically removes the container when it stops. This is useful for running temporary or one-time containers without cluttering your Docker environment with stopped containers.
+
+To build the image locally, run
+
+```bash
+docker build -t sleap-chrome-remote-desktop .
+```
+
+Then, to run the image with gpus interactively
+
+```bash
+docker run --rm -it --gpus=all sleap-chrome-remote-desktop
+```
+
+# Connecting to the Chrome Remote Desktop
+
+This guide explains how to connect to the Chrome Remote Desktop server running in the container to access the graphical user interface (GUI).
+
+## Prerequisites
+
+1. **Google Account**: You need a Google account to use Chrome Remote Desktop.
+2. **Chrome Browser**: You need to have the Chrome browser installed on your local machine.
+3. **Docker Setup**: 
+   - Ensure Docker is installed on your local machine.
+
+## Steps to Connect to the Chrome Remote Desktop
+
+### 1. Start the Docker Container
+
+Start the Docker container with the Chrome Remote Desktop server running in it.
+
+```bash
+docker run --rm -it --gpus=all <your-container-name> bash
+```
+
+### 2. Connect to the Chrome Remote Desktop
+
+1. Go to [https://remotedesktop.google.com/access](https://remotedesktop.google.com/access) in your Chrome browser.
+
+2. Sign in with your Google account.
+
+3. Then, click "Set up via SSH" on the left and follow the provided steps.
+- This will provide the command in this form: 
+  ```bash
+  DISPLAY= /opt/google/chrome-remote-desktop/start-host \
+    --code="4/xxxxxxxxxxxxxxxxxxxxxxxx" \
+    --redirect-url="https://remotedesktop.google.com/_/oauthredirect" \
+    --name=$(hostname)
+  ```
+
+1. Run the provided command in your terminal with Docker container running. Make sure to refresh the website. 
+
+2. When prompted, enter 6-digit PIN. This number will be used for additional authorization when you connect later.
+
+### 3. Run Chrome Remote Desktop
+
+1. Go to [https://remotedesktop.google.com/access](https://remotedesktop.google.com/access) in your Chrome browser.
+
+2. Sign in with your Google account.
+
+3. Then, click "Remote Access" to check the status of your remote desktop (make sure to refresh the page).
+
+4. If you see the status as "Online", click on the remote desktop to connect and enter the 6-digit PIN you set earlier.
+
+### 4. Access the GUI
+
+Once connected, you'll see the desktop environment configured in the container. You can now open and use GUI-based applications such as `sleap-label`.
```

Comment:
open a terminal and use sleap commands like `sleap-label`. The GUI should be displayed in your chrome remote desktop display environment on your local machine.